### PR TITLE
Improve script for TiDB tests

### PR DESCRIPTION
**Pull request by borissavelev**
* use isolated docker network instead of magic with MYIP
* init script can accept TiDB version
* minor changes in usage help

### CODE REVIEW

**Comment by boltandrke** [comment]
Maybe its only my computer issue, but it fails during TiDB acceptance tests


`make testtidb6.1.0
/Library/Developer/CommandLineTools/usr/bin/make MYSQL_VERSION=6.1.0 MYSQL_PORT=34610 testtidb
==> Pulling up TiDB cluster with TiKV and TB components
==> Removing any existing TiDB cluster components
64e0cb3dbefcfc7a3e59cfa9ef0ed09fa4d198d7eea6533fbea30e82f1d7193e
==> Pulling up PD component
==> Pulling up TiKV component
==> Pulling up TiDB component
Waiting for TiDB...
....................................................................................................................................................................................................................................................................................
`

Looks like TiKV  will exit immediately after startup

`
docker ps -a
CONTAINER ID   IMAGE                 COMMAND                  CREATED          STATUS                       PORTS                                                     NAMES
485fbc0d90fc   pingcap/tidb:v6.1.0   "/tidb-server --stor…"   8 seconds ago    Up 3 seconds                 4000/tcp, 0.0.0.0:34610->34610/tcp, :::34610->34610/tcp   tidb
f1397129df1b   pingcap/tikv:v6.1.0   "/tikv-server --addr…"   11 seconds ago   Exited (132) 6 seconds ago                                                             tikv
1103054bfafa   pingcap/pd:v6.1.0     "/pd-server --name=p…"   15 seconds ago   Up 10 seconds                2379-2380/tcp                                             pd
`

TiKV logs:

`....
ool-size\":2},\"resource-metering\":{\"receiver-address\":\"\",\"report-receiver-interval\":\"1m\",\"max-resource-groups\":100,\"precision\":\"1s\"},\"causal-ts\":{\"renew-interval\":\"100ms\",\"renew-batch-min-size\":100}}"]
[2022/08/22 07:14:03.848 +00:00] [WARN] [server.rs:457] ["failed to remove space holder on starting: No such file or directory (os error 2)"]
[2022/08/22 07:14:03.851 +00:00] [INFO] [mod.rs:124] ["encryption: none of key dictionary and file dictionary are found."]
[2022/08/22 07:14:03.852 +00:00] [INFO] [mod.rs:485] ["encryption is disabled."]
[2022/08/22 07:14:03.877 +00:00] [INFO] [engine.rs:91] ["Recovering raft logs takes 13.062ms"]
`

File: scripts/tidb-test-cluster.sh [hunk 1]
```diff
@@ -12,11 +13,12 @@ fi
 SCRIPT_PATH=$(realpath $(dirname "$0"))
 TEMPDIR=$(mktemp -d)
 DOCKER=$(which docker)
-TAG_VERSION="v6.1.0"
 SCRIPT_INIT=false
 MYIP="127.0.0.1"
```
Comment (petoju) [inline scripts/tidb-test-cluster.sh:17]: Please remove also `MYIP` as it's not longer used (if I'm reading it correctly).

**Comment by petoju** [comment]
You should try running `make acceptance` and it should pass.
Thanks @boltandrke for noticing that.

**Comment by borissavelev** [comment]
> You should try running `make acceptance` and it should pass. Thanks @boltandrke for noticing that.
I did... Looks like some issues on M1. I reverted tikv image tag

**Comment by borissavelev** [comment]
sorry, typo found 🤦 

all green now

**Pull request merged**